cylc/flow/batch_sys_handlers/loadleveler.py

```diff
@@ -15,6 +15,8 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """Submits task job scripts to loadleveler by the ``llsubmit`` command.
 
+# TODO: rewrite the following for platforms:
+
```

Comment:
Presumably these code block examples need to be updated from the `[runtime][X][job]` to `[platforms]`?

Reply:
Yep, https://github.com/cylc/cylc-flow/issues/3997